### PR TITLE
Adjust SDK reference, add some API's, bump 1.1.0

### PR DESCRIPTION
**Pull request by hansemannn**
Hey @williamrijksen, some minor changes and two new methods `getTags` and `setLogLevel`. Untested for now, so please review before, thx!

### CODE REVIEW

File: README.md
Comment (williamrijksen) [inline README.md]: Please remove this.

File: README.md
Comment (williamrijksen) [inline README.md:56]: Unfortunately this function runs into a crash.

`Service exited due to signal: Segmentation fault: 11`

Please fix before i'll merge.

File: README.md
Comment (williamrijksen) [inline README.md:64]: Is it possible to add the `onesignal.setLogLevel(onesignal.LOG_LEVEL_DEBUG);` as well into the readme? I had a look into the cordova plugin i like that syntax. Is it possible to change that function into this?

```
setLogLevel({logLevel: onesignal.LOG_LEVEL_DEBUG, visualLevel: onesignal.LOG_LEVEL_DEBUG});
```

https://documentation.onesignal.com/docs/phonegap-sdk#section--setloglevel-

File: iphone/titanium.xcconfig
Comment (williamrijksen) [inline iphone/titanium.xcconfig:7]: Why did you upgrade this version?

**Review by williamrijksen — APPROVED** [review]
Unfortunately the `getTags` runs into a segmentation fault. Please fix before I'll merge.

File: iphone/Classes/ComWilliamrijksenOnesignalModule.m
Comment (williamrijksen) [inline iphone/Classes/ComWilliamrijksenOnesignalModule.m:107]: Nice!

**Review by williamrijksen — CHANGES_REQUESTED** [review]

**Comment by williamrijksen** [comment]
@hansemannn Thanks for you're contributions but i can't merge this unfortunately. Please fix.

File: iphone/titanium.xcconfig
Comment (hansemannn) [reply iphone/titanium.xcconfig:7]: It's the latest patch SDK, so it's recommended to use that when building the module - this does not change the minimum SDK

File: README.md
Comment (hansemannn) [reply README.md:64]: Sure thing

File: README.md
Comment (hansemannn) [reply README.md]: Why? It enables syntax-highlighting (e.g. https://github.com/hansemannn/ti.googlemaps and https://github.com/hansemannn/ti.paypal)

Comment (williamrijksen) [reply README.md]: Because it looks like this

schermafbeelding 2016-11-10 om 13 07 21

File: iphone/titanium.xcconfig
Comment (williamrijksen) [reply iphone/titanium.xcconfig:7]: Okay 👍

**Comment by hansemannn** [comment]
PR updated

**Review by williamrijksen — APPROVED** [review]

**Comment by williamrijksen** [comment]
Verified!

**Comment by williamrijksen** [comment]
@hansemannn Introduces a new release for this: https://github.com/williamrijksen/com.williamrijksen.onesignal/releases/tag/iOS-1.1.0.

Thank you so much!

**Pull request merged**
